internal/keyvault/keyvault.go

```diff
@@ -17,29 +17,27 @@ import (
 	"github.com/Azure/notation-azure-kv/internal/crypto"
 )
 
-// clientAuthMode is the authorize mode used by plugin
-type clientAuthMethod string
-
 // set of auth errors
 var (
 	errMsgAuthorizerFromMI  = "authorized from Managed Identity failed, please ensure you have assigned identity to your resource"
 	errMsgAuthorizerFromCLI = "authorized from Azure CLI 2.0 failed, please ensure you have logged in using the 'az' command line tool"
 	errMsgUnknownAuthorizer = "unknown authorize method, please use a supported authorize method according to the document"
 )
 
+// authorizer is the authorize mode used by plugin
+type authorizer string
```

Comment:
Should we call it `authMode` or `authMethod`?

Reply:
Updated to use `authMethod`